doc/classes/Object.xml

```diff
@@ -53,62 +64,80 @@
 		<method name="_init" qualifiers="virtual">
 			<return type="void" />
 			<description>
-				Called when the object is initialized in memory. Can be defined to take in parameters, that are passed in when constructing.
-				[b]Note:[/b] If [method _init] is defined with required parameters, then explicit construction is the only valid means of creating an Object of the class. If any other means (such as [method PackedScene.instantiate]) is used, then initialization will fail.
+				Called when the object's script is initialized in memory. Can be defined to take in parameters. This method is similar to a constructor in most programming languages.
```

Comment:
I was reluctant to introduce the concept of script instances, with no explanation in the leading description. From the user's perspective, there's little need to know about it, because it's easier to think of the Script as a Resource that can be used to extend Object over Script Vs. Script Instance. It doesn't mean I was going to strip away every mention of it, but it requires some thought.